<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportInventoryWidget</class>
 <widget class="QWidget" name="ImportInventoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>119</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&amp;Quantity:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="buddy">
        <cstring>w_qty</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="w_qty">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Condition:</string>
       </property>
       <property name="buddy">
        <cstring>w_condition_new</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="w_condition_new">
         <property name="text">
          <string>&amp;New</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="w_condition_used">
         <property name="text">
          <string>&amp;Used</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>E&amp;xtra parts:</string>
       </property>
       <property name="buddy">
        <cstring>w_extra</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="w_extra">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string extracomment="Extra parts state">Include</string>
        </property>
        <property name="icon">
         <iconset theme="vcs-normal">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exclude</string>
        </property>
        <property name="icon">
         <iconset theme="vcs-removed">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra</string>
        </property>
        <property name="icon">
         <iconset theme="vcs-added">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string extracomment="Label on the left">Include:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_instructions">
       <property name="text">
        <string>&amp;Instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_originalBox">
       <property name="text">
        <string>Original Box</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_alternates">
       <property name="text">
        <string>&amp;Alternates</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_counterParts">
       <property name="text">
        <string>&amp;Counterparts</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Also part out:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_setsInSet">
       <property name="text">
        <string>Sets in set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="w_minifigs">
       <property name="text">
        <string>Minifigs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
